bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -508,6 +528,42 @@
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 	</channel-type>
 
+	<channel-type id="armed">
+		<item-type>Switch</item-type>
+		<label>@text/channelAlarmArmedLabel</label>
+		<description>@text/channelAlarmArmedDescription</description>
+		<category>Alarm</category>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
```

Comment:
For my curiosity, why setting this policy ?

Reply:
When you arm, it will take e.g. 30 seconds before it is fully armed, typically the time to leave the home. The armed state should therefore not be set directly by the UI, but only after 30s when there is a feedback from the alarm system it is effectively armed. The veto policy makes sure to only set based on what comes back from the device, not use the initial UI set value.

Reply:
Makes sense.